site/_components/banner-interstitial.njk

```diff
@@ -0,0 +1,34 @@
+<!--{#
+  Renders a full-width banner that transitions to a contained banner. Requires a wrapper element with the class `banner-interstitial__wrapper`.
+
+  @param {object} content: An object containing the following attributes.
+  @param {string} content.eyebrow
+  @param {string} content.title
+  @param {string} content.copy
+
+  @param {object} image
+  @param {string} image.src
+  @param {string} image.alt
+
+  @param {object} cta (optional): An object containing the following attributes.
+  @param {string} cta.text
+  @param {string} cta.url
+  
+  @param {object} modifiers (optional): An object containing the following attributes.
+  @param {string} modifiers.class: Modifiers class/classes
+#}-->
+{% from 'icon.njk' import icon %} {% macro bannerInterstitial(content, image, cta, modifiers) %}
+<article class="banner-interstitial {{'banner-interstitial--no-cta' if not cta}}{% if modifiers.class %} {{modifiers.class}} {% endif %}">
+  <div class="banner-interstitial__content">
+    <div class="banner-interstitial__text">
+      <small class="banner-interstitial__eyebrow type--h1">{{content.eyebrow}}</small>
+      <h2 class="banner-interstitial__title type--mega">{{content.title}}</h2>
+      <p class="banner-interstitial__copy type--base">{{content.copy}}</p>
+      {% if cta is defined %}<a href="{{cta.url}}" class="cta cta--transparent banner-interstitial__cta"> {{cta.text}} </a>{% endif %}
```

Comment:
Same `locale` comment as in card. As this keeps coming up, it may be worthwhile creating a filter for it to capture the logic; it'd look something like:

```
{{cta.url | relative(locale)}}
```

(or something like that, would be great if it could be `locale` too).

The logic would be something like:
* If `locale` exists and it's a relative URL _and_ it isn't already prefixed with a locale, prepend the locale code
* Return URL